docs/manual/developer/03_creating_content.md

```diff
@@ -558,6 +558,61 @@ controls:
       and to use the targeted policy.
 ```
 
+### Reporting status
+
+In some cases, it's useful to know the status of a certain control for a
+specific product. In order to better portray this, it's possible to set
+such information on each control using the `status` key.
+
+The `status` key may hold the following values:
+
+* `pending`: The control is not yet evaluated for the product.
+
+* `not applicable`: The control is not applicable to this product.
+
+* `inherently met`: The control is inherently met by the product.
+
+* `documentation`: The control is addressed by product documentation.
+
+* `planned`: The control is not yet implemented, but is planned.
+
+* `partial`: While work has been done to address this control, there is still
+             work needed to fully address it.
+
+* `complete`: The control is addressed by the product.
+
+When there is work on-going to address a specific control, it may be portrayed
+via the `tickets` key. The aforementioned key shall contain a list of URLs that
+may help the reader track what work needs to be done to address a specific
+control.
```

Comment:
To understand a bit more the nature of the tickets. Would they be about features that need to be implemented in the product to be able to satisfy a control, or about the actual writing of rules, or any kind of support in CaC? Or both?

Reply:
I was thinking more along the lines of "features that need to be implemented in the product to be able to satisfy a control"

Reply:
It is already quite trivial for us to determine if a control is automated since... well, this is in the same repo as the automation for the control. So all we need to do is build the content, check the rules pertaining to the control it have OVAL and that's it.